data/json/snippets/mi-go_anatomy.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "type": "snippet",
+    "category": "<mi-go_harvest>",
+    "text": [
+      "As you starts to cut through the thick skin if the mi-go, you note that it's tougher than you would have expected.",
+      "The smell emanating from the mi-go as you cut it open is pungent and very alien.  You're forced to take short breaks to recover from nausea.",
+      "You note that the blood of this creature is unlike any other type of blood you've ever encountered or even heard of.  It's not only the odd color, but also its viscosity and somewhat oily and sticky properties.",
+      "Nothing seems right with this creature.  The texture of the muscles is off.",
+      "You note that the bones in this body have alien shapes and are joined up in ways unlike anything from Earth.",
+      "What should go for tendons in this creature aren't joined to the bones in any way you've seen before.  Instead of a single tendon connecting two end points, there are instead a range of smaller connections between slightly different locations."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<mi-go_harvest_deeper>",
+    "text": [
+      "You cannot find a single heart in the mi-go.  Instead, there seem to be multiple copies of something that presumably acts as a heart spread out all over the body.  At least you think it acts as a heart, given how it is connected to what should be blood vessels.",
+      "Something that looks like it should be blood vessels, on account of containing the mi-go's blood, form a network that doesn't make much sense to you, as its structure is completely alien.",
+      "The creature contains a lot of tissue that probably are organs of one kind of another, but there is no guessing what function they are meant to perform.",
+      "The mi-go doesn't seem to have any known type of lungs, but there are a lot of small orifices all over the body connecting short sections of tubing to organs of spongy tissue, so it might take in air (or water?) through one orifice, extract oxygen from the medium, and \"exhale\" it through the tubing at the other side of the organ.  However, you can not detect any means to actually drive such a process.",
+      "You think you have found the creature's voice box, but you cannot be sure, given its design which leaves you unable to figure out how it would actually work.",
```

Comment:
```suggestion
      "You think you have found the creature's voice box, but you cannot be sure, given its design which leaves you unable to figure out how it would actually work.  It reminds you a bit of an old-fashioned bellows, if a bellows were sky-blue, spongy, and smelled like rotten ammonia.",
```